tests/pytests/unit/modules/test_pcs.py

```diff
@@ -0,0 +1,314 @@
+    exp_cmd = (
+        exp_cmd + [test_data.cluster_name] + test_data.nodes + test_data.extra_args
+    )
+
+    mock_cmd = MagicMock()
+    patch_salt = patch.dict(
+        pcs.__salt__,
+        {
+            "cmd.run_all": mock_cmd,
+            "pkg.version_cmp": MagicMock(return_value=ver_cmp_ret),
+        },
+    )
+    
+    with patch_salt:
+        pcs.cluster_setup(
+            test_data.nodes, test_data.cluster_name, extra_args=test_data.extra_args
+        )
+    assert mock_cmd.call_args_list[0][0][0] == exp_cmd
+
+def test_cluster_destroy(test_data):
+    """
+    Test for destroying a cluster
+    """
+    exp_cmd = ["pcs", "cluster", "destroy"]
+    exp_cmd.extend(test_data.extra_args)
+
+    mock_cmd = MagicMock()
+    patch_salt = patch.dict(pcs.__salt__, {"cmd.run_all": mock_cmd})
+    with patch_salt:
+        pcs.cluster_destroy(
+            extra_args=test_data.extra_args
+        )
+    
+    assert mock_cmd.call_args_list[0][0][0] == exp_cmd
+
+def test_cluster_node_add(test_data):
+    """
+    Test for adding a cluster
+    """
+    exp_cmd = ["pcs", "cluster", "node", "add"]
+    exp_cmd = exp_cmd + [test_data.nodea] + test_data.extra_args
+
+    mock_cmd = MagicMock()
+    patch_salt = patch.dict(pcs.__salt__, {"cmd.run_all": mock_cmd})
+    with patch_salt:
+        pcs.cluster_node_add(test_data.nodea, extra_args=test_data.extra_args)
+    assert mock_cmd.call_args_list[0][0][0] == exp_cmd
+
+def test_cib_push(test_data):
+    """
+    Test for pushing a CIB file
+    """
+    exp_cmd = ["pcs", "cluster", "cib-push", test_data.cib_filename, 'scope=configuration']
+    exp_cmd.extend(test_data.extra_args)
+
+    mock_cmd = MagicMock()
+    patch_salt = patch.dict(pcs.__salt__, {"cmd.run_all": mock_cmd})
+    with patch_salt:
+        pcs.cib_push(
+            test_data.cib_filename, scope="configuration", extra_args=test_data.extra_args
+            )
+    assert mock_cmd.call_args_list[0][0][0] == exp_cmd
+
+def test_config_show(test_data):
+    """
+    Test for config show
+    """
+    exp_cmd = ["pcs", "cluster", "cib-push", test_data.cib_filename, 'scope=configuration']
+    exp_cmd.extend(test_data.extra_args)
+
+    mock_cmd = MagicMock()
+    patch_salt = patch.dict(pcs.__salt__, {"cmd.run_all": mock_cmd})
+    with patch_salt:
+        pcs.cib_push(
+            test_data.cib_filename, scope="configuration", extra_args=test_data.extra_args
+            )
+    # print("Exp command: ", exp_cmd)
+    # print("call_args_list: ", mock_cmd.call_args_list[0][0][0] )
+    assert mock_cmd.call_args_list[0][0][0] == exp_cmd
+
+
+def test_item_show_defaults(test_data):
+    """
+    Test for item show
+    """
+    exp_cmd = ["pcs", ]
+    exp_cmd.extend(test_data.extra_args)
+
+    mock_cmd = MagicMock()
+    patch_salt = patch.dict(pcs.__salt__, {"cmd.run_all": mock_cmd})
+    with patch_salt:
+        pcs.cib_push(
+            test_data.cib_filename, scope="configuration", extra_args=test_data.extra_args
```

Comment:
I'm not seeing where you are testing item show defaults here.

Reply:
Me too. Deleted it.